lib/jxl/dec_frame.cc

```diff
@@ -797,24 +812,25 @@ Status FrameDecoder::ProcessSections(const SectionInfo* sections, size_t num,
           return PrepareStorage(num_threads,
                                 decoded_passes_per_ac_group_.size());
         },
-        [this, &ac_group_sec, &num_ac_passes, &num, &sections, &section_status,
-         &has_error](size_t g, size_t thread) {
-          if (num_ac_passes[g] == 0) {  // no new AC pass, nothing to do.
+        [this, &ac_group_sec, &desired_num_ac_passes, &num, &sections,
+         &section_status, &has_error](size_t g, size_t thread) {
+          if (desired_num_ac_passes[g] ==
+              0) {  // no new AC pass, nothing to do.
```

Comment:
Can you move the comment one line up?

Reply:
sure

Reply:
actually the linter didn't allow it, moving it one line below.